hack/api.md

```diff
@@ -87,7 +87,7 @@ the following hierarchy:
         └── src
             └── main
                 └── java
-                    └── org.apache.cloudstack
+                    └── org/apache/cloudstack
```

Comment:
@rudimk Most Java devs get this - these are package names which are separated by `.` and depending on operation system (windows vs unix-like), these are indeed folders like you've mentioned. 

You may refer more here:
https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html
https://stackoverflow.com/questions/60867518/java-package-folder-naming-conventions